Example/Backpack.xcodeproj/project.pbxproj

```diff
@@ -776,11 +777,6 @@
 				BUNDLE_LOADER = "$(TEST_HOST)";
 				CLANG_WARN_DEPRECATED_OBJC_IMPLEMENTATIONS = YES;
 				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = YES;
-				FRAMEWORK_SEARCH_PATHS = (
-					"$(SDKROOT)/Developer/Library/Frameworks",
-					"$(inherited)",
-					"$(DEVELOPER_FRAMEWORKS_DIR)",
-				);
```

Comment:
This is to fix this error:

![screen shot 2018-09-12 at 09 53 36](https://user-images.githubusercontent.com/73652/45413726-bcce5a80-b671-11e8-8403-4ec7ee60b804.png)